subsys/bluetooth/mesh/sensor.c

```diff
@@ -475,7 +475,7 @@ int sensor_cadence_decode(struct net_buf_simple *buf,
 	}
 
 	delta_format = (threshold->delta.type == BT_MESH_SENSOR_DELTA_PERCENT) ?
-			       &bt_mesh_sensor_format_percentage_delta_trigger :
+			       &bt_mesh_sensor_format_percentage_16 :
```

Comment:
The separate definition for this type was added to allow the delta to go above 100%. Is this no longer allowed by the test spec?

Reply:
But was it allowed before or is it just missing definition in the spec? Although, it sounds reasonable to allow setting delta trigger >100%.

Reply:
I was under the impression that a qualification test was failing because we didn't process the >100% scenario correctly. Perhaps @michal-narajowski has some details?

Reply:
PTS sends this Sensor Cadence Set message: `0x82FFFFFFFF0A0840F8BF`, where Status Trigger Type is 1 (percentage) and Delta Up/Down are 0xFFFF. By looking at other definitions for percentage format, it looks like PTS treats 0xFFFF as `value is not known`. MshMDL doesn't say anything except this:

>The value of 0b1 means that the unit is «unitless», the format type is 0x06 (uint16), and the value is represented as a percentage change with a resolution of 0.01 percent. 

Test spec refers to Mesh Device Properties document, which doesn't define format for this case.

Reply:
Alright, seems very reasonable.